transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/EncodeTransformationPipelineBuilder.scala

```diff
@@ -33,11 +35,9 @@ object EncodeTransformationPipelineBuilder extends PipelineBuilder[Args] {
   override def buildPipeline(ctx: ScioContext, args: Args): Unit = {
     // read in extracted info
     val donorInputs = StorageIO
-      .readJsonLists(
-        ctx,
-        "Donors",
-        s"${args.inputPrefix}/Donor/*.json"
-      )
+      .readJsonLists(ctx, "Donors", s"${args.inputPrefix}/Donor/*.json")
+    val antibodyInputs = StorageIO
+      .readJsonLists(ctx, "AntibodyLots", s"${args.inputPrefix}/AntibodyLot/*.json")
```

Comment:
Sorry to sneak in merge conflicts 🙁 there should be a new helper method that you can use to read these while stripping out unknown values.